docs/jobs.md

```diff
@@ -0,0 +1,432 @@
+Cozy Jobs
+=========
+
+Jobs are designed to represent asynchronous tasks that your cozy can execute. These tasks can be scheduled in advance, recurring or suddenn and provide various services.
+
+At the time, we do not provide "generic" and programmable tasks. This list of available workers is part of the defined API.
+
+The job queue can be either local the cozy-stack when used as a monolitic instance (self-hosted for instance) or distributed via a redis server for distributed infrastructures.
+
+This doc introduces two cozy types:
+
+  - `io.cozy.jobs` for jobs
+  - `io.cozy.launchers` for launchers
+
+
+Launchers
+---------
+
+Jobs can be launched by three types of launchers:
+
+  - `@cron` to schedule recurring jobs, either periodic or scheduled at specific times
+  - `@interval` to schedule jobs executed at a given fix interval
```

Comment:
My first impression is that `@cron` and `@interval` are 2 ways to do the same thing. What are the differences between them?